tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -82,6 +83,37 @@ def test_arxiv_fulltext_download_logs_on_success():
 
 
 @pytest.mark.httpretty
+def test_arxiv_fulltext_download_not_ready():
+    httpretty.register_uri(
+        httpretty.GET, 'http://export.arxiv.org/pdf/1605.03844',
+        body=pkg_resources.resource_string(
+            __name__, os.path.join('fixtures', 'notreadyblob.gz')))
```

Comment:
In order to prove the point of _retrying_ we can in fact provide two responses: the first one fails, the second one works.